packages/graphql-transformer-common/src/definition.ts

```diff
@@ -16,13 +16,11 @@ import {
   DirectiveNode,
   EnumTypeDefinitionNode,
   ValueNode,
-  ListValueNode,
-  ObjectValueNode,
   InputObjectTypeDefinitionNode,
 } from 'graphql';
 
 type ScalarMap = {
-  [k: string]: 'String' | 'Int' | 'Float' | 'Boolean' | 'ID';
+  [k: string]: 'String' | 'Int' | 'Float' | 'Boolean' | 'ID' | 'Date';
```

Comment:
Since this list is the same as the inputs list above, would it be better to pull this into a string enum? This is similar to what I did with the enum in my auth PR. Based on [this issue](https://github.com/microsoft/TypeScript/issues/13542) it seems to be the recommended way to have a string union type and be able to enumerate the type at runtime

Reply:
That would be a good add in but I think the lists for scalars are not indicative of the input types for `@searchable`.